tests/sles4sap/sapconf.pm

```diff
@@ -35,6 +35,52 @@ sub check_profile {
       unless ($output =~ /Current active profile: $profile/);
 }
 
+sub run_developers_tests {
+    my $devel_repo = 'https://gitlab.suse.de/AngelaBriel/sapconf-test/repository/master/archive.tar.gz';
+    my $log        = '/tmp/sapconf_test.log';
+
+    # Download and unpack the test scripts supplied by the developers
+    # Record soft failure and continue if it can not be downloaded
+    my $ret    = script_run "curl -k $devel_repo > /tmp/sapconf-test.tar.gz";
+    my $output = script_output "file /tmp/sapconf-test.tar.gz";
```

Comment:
curl will return non-zero exit code in case of any problem , right ? not sure why you need to check something that you already know.

Reply:
gitlab loves to throw login prompts in HTML at you - not a problem to curl, but to tar

Reply:
make sense

Reply:
As @coolo said. curl can return 0 and leave a non gzip file in `/tmp/sapconf-test.tar.gz` ... basically a bunch of HTML data.